test/user.bats

```diff
@@ -86,6 +86,13 @@ setup() {
         assert_output "$altered_mamba_user"
 }
 
+# Test that home is moved when modifying the username.
+@test "docker run --rm -e MAMBA_USER=$altered_mamba_user ${MICROMAMBA_IMAGE}-modify-username bash -c \"realpath ~${altered_mamba_user}\"" {
+        run docker run --rm -e MAMBA_USER=$altered_mamba_user ${MICROMAMBA_IMAGE}-modify-username bash -c "realpath ~${altered_mamba_user}"
```

Comment:
Add quotes to guard against  variables containing spaces:

```suggestion
        run docker run --rm -e "MAMBA_USER=$altered_mamba_user" "${MICROMAMBA_IMAGE}-modify-username" bash -c "realpath ~${altered_mamba_user}"
```

Reply:
Thanks. For my information, this is _purely_ on principle, right? Firstly, valid usernames and image names cannot contain spaces. Secondly this is for tests where we control the inputs.

It's awkward to escape the quotes in the `@test` line.

Reply:
Okay, I manged to find your comment again. Where you put it is fine, I just didn't realize that review comments get collapsed within the PR conversation. I've been trying to make use of the Github review functionality more, but obviously I'm still learning some of the details.